optaplanner-operator/src/k8s/example-solver.yml

```diff
@@ -0,0 +1,10 @@
+# Example solver resource; override as you see fit.
```

Comment:
File naming: should we name this school-timetabling-solver.yml ?

Reply:
Good point; either that or I will make the example more general, without referring to any particular use case.